SecurityAgent/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.1.1] - 2023-06-20
+### Changed
+- Update Thunder Security Logging to add to Telemetry
+
 ## [1.1.1] - 2023-06-05
```

Comment:
Hi @anand-ky Here already 1.1.1 there and API_VERSION_NUMBER_PATCH as 0 .
Now I neeed to update here 1.1.2 and API_VERSION_NUMBER_PATCH as 2 ?

Reply:
yes pls update to 1.1.2

Reply:
Done anand